kotest-assertions/kotest-assertions-core/src/jvmMain/kotlin/io/kotest/matchers/reflection/compose.kt

```diff
@@ -1,28 +0,0 @@
-package io.kotest.matchers.reflection
-
-import io.kotest.matchers.Matcher
-import io.kotest.matchers.MatcherResult
-import kotlin.reflect.KProperty1
-
-@Suppress("UNCHECKED_CAST")
-fun <T : Any?> Matcher.Companion.compose(
```

Comment:
Maybe we should keep this one with a deprecation and replacewith pointing to the new location (`io.kotest.matchers.compose`)

Since the package changes, imports will stop working without notice otherwise. We can remove this one in 6.0 I think